src/services/ui/src/pages/dashboard/index.tsx

```diff
@@ -63,8 +64,8 @@ export const Dashboard = () => {
           <div className="flex items-center justify-between my-4">
             <h1 className="text-xl">Dashboard</h1>
             {!userContext?.isCms && (
-              <Button>
-                <Link to={ROUTES.NEW_SUBMISSION_OPTIONS}>New Submission</Link>
+              <Button onClick={() => navigate(ROUTES.NEW_SUBMISSION_OPTIONS)}>
```

Comment:
The `<Button>` element is typically used for actions on the same page, like submitting a form or toggling a menu. Using it for navigation can be confusing for screen reader users. 

Using a `<Link>` (which intrinsically renders `<a>`) element styled as a button for navigation purposes. This ensures that the element is semantically correct and more predictable for assistive technologies.

Let's change this to keep our semantics right, and drop the button entirely. You should be able to style the `<Link>` as a button using `className`.